### ISSUE TITLE
Could I get the ImageNet-1K pre-training log? 

### ISSUE THREAD

**Issue by hyj378** [LayneH/LEWEL#2]
Hello. Thank you for your fantastic work.

I trained the model on ImageNet-1K through modified `scripts/run_lewelb_eman.sh`  and I got NN ACC@1 score of 19.376 after 124 epochs training.  But When I evaluate using the provided checkpoint(`lewel_b_100ep.pth`) with same validation setting, I got NN ACC@1 score of 35.004.

I think the training has progressed due to loss, so I want to find my fault by comparing the original log.
If possible, could I get the training log on ImageNet-1K pre-training?

![image](https://user-images.githubusercontent.com/41140561/195337841-32cfa4e7-102e-40df-80f4-88c07c720a13.png)

**Comment by LayneH** [comment]
Hi,
You can find the pretraining log [here](https://github.com/LayneH/LEWEL/blob/main/pretrain_logs/LEWEL_B-100ep.txt).

**Comment by hyj378** [comment]
Oh, Thank you I appreciate it

I found the difference between mine and your log. the dataset iterations number is different. I think the total iterations(1251 )Xbatch size(256) is not same the ImageNet 1K number(1,281,024). Could I have missed something?

And I have some questions about the training parameter. I guess the setting is different from the paper. 

In the paper: 
`For LEWELB, we use the batch size of 512, initial learning rate of 1.8, weight decay of 10−6 , and set the initial momentum α of the encoder to 0.98, which is increased to 1 according to the cosine schedule`
In `run_lewelb_eman.sh`:
```
LR=3.6
BS=1024
WD=0.0000015
BETA=0.98
EP=200
WARM_EP=10
LEWEL_SCALE=1.0
LEWEL_NUM_HEADS=4
LEWEL_LOSS_WEIGHT=0.5
```

If there are something wrong, please let me know.

**Comment by hyj378** [comment]
I found what is different. thank you.
